indico/modules/events/templates/header.html

```diff
@@ -38,15 +38,20 @@
        title="{% trans %}Change theme{% endtrans %}"></a>
     <ul class="dropdown">
         {% set endpoint = 'timetable.timetable' if event.type == 'conference' else 'events.display' %}
-        {% for id, data in themes.items()|sort(attribute='1.name')
-           if event.can_manage(session.user) or data.user_visible %}
-            <li>
-                {% if id == theme %}
-                    <a class="disabled"><strong>{{ data.name }}</strong></a>
-                {% else %}
-                    <a href="{{ url_for(endpoint, event, view=id) }}">{{ data.name }}</a>
-                {% endif %}
-            </li>
+        {% for grouper, list in themes.items()|sort(attribute='1.name')|groupby('1.user_visible')|reverse
```

Comment:
I was more thinking about `for group in ...` and then using `group.grouper` and `group.list` in the code below

Reply:
FWIW, if you want to keep the argument unpacking style, maybe go for `user_visible` and `themes_list`... the latter is not very nice naming but at least I can't think of anything better..